.github/workflows/main-ci.yml

```
@@ -0,0 +1,51 @@
+name: Lint & install charts
```

```suggestion
name: Lint and test chart installation
```
Would this clarify the meaning of this workflow displayed in pull request checks